src/submission/submission-service.ts

```diff
@@ -867,8 +867,12 @@ export namespace operations {
     }
 
     if (mutableClinicalData.length > 0) {
+      const msg = expectedClinicalEntites.includes(ClinicalEntitySchemaNames.REGISTRATION)
```

Comment:
like we discussed this logic should probably live in `submission-error-messages.ts`